deploy/src/erc_to_erc/foreign.js

```diff
@@ -71,8 +73,6 @@ async function deployForeign() {
     url: FOREIGN_RPC_URL
   });
   assert.equal(txInitializeForeign.status, '0x1', 'Transaction Failed');
-  const validatorOwner = await bridgeValidatorsForeign.methods.owner().call();
```

Comment:
Just to write it somewhere: as we discussed on the call this issue could appear in situation when a block which includes the transaction to change the state of the contract become an uncle. In this case the state introduced by the transaction will not be changed until the transaction is included in a new block. 

We remove the call since
a) there is no need get owner of the contract since it is used just to display the owner address, but it is known any way;
b) there is no simple way without significant delay (to wait some number of blocks to finalize state) to fix this issue.

Since the issue common for all chains and to have user interface consistency please consider to remove similar code from:
1. `erc_to_erc/home.js`
2. `native_to_erc/foreign.js`
3. `native_to_erc/home.js`

Reply:
will do